README.md

```diff
@@ -1,36 +1,315 @@
 # Sparrow
+## Summary
+Sparrow is an Elixir library for [push notifications](en.wikipedia.org/wiki/Push_technology#Push_notification).
+[![Build Status](https://travis-ci.org/esl/sparrow.svg?branch=master)](https://travis-ci.org/esl/sparrow)
+[![Coverage Status](https://coveralls.io/repos/github/esl/sparrow/badge.svg)](https://coveralls.io/github/esl/sparrow)
 
-**TODO: Add description**
+Currently it provides support for the following APIs:
+* [FCM v1](https://firebase.google.com/docs/cloud-messaging/)
+* [APNS](https://developer.apple.com/library/archive/documentation/NetworkingInternet/Conceptual/RemoteNotificationsPG/APNSOverview.html)
 
-**TODO: Redirect Erlang lager to Elixir logger**
+## Requirements
+* Elixir 1.6 or higher
+* Erlang OTP 20.3.6 or OTP 21
 
-**TODO: Add publishing package to hex** 
+# Build sparrow config
+This section describes how to write a config file for Sparrow.
+If you wish to use just one of the following services, do not include the other in the config.
 
-**TODO: Check API diffs between these articles:** 
+```elixir
+config :sparrow, config: [fcm: [...], apns: [...]]
```

Comment:
Honestly, I'm still a bit at a loss as to why the config is nested under `:config`. It stands out more now that it's spelled out in the readme.

Reply:
how about:
```elixir
config :sparrow, fcm: 
[...]

config :sparrow, apns: 
[...]
```
??

Reply:
Normally it's just:

```elixir
config :sparrow,
    fcm: [
        ...
    ],
    apns: [
        ...
    ]
```

Reply:
something like: github.com/esl/sparrow/pull/5/ ??